README.md

```diff
@@ -269,27 +269,27 @@ Using AWS's 8 core server, APISIX's QPS reach to 140,000 with a latency of only
-| Latency                                                         | 0.2 ms                                            | 2 ms                    |
-| Dubbo                                                           | Yes                                               | No                      |
-| Configuration rollback                                          | Yes                                               | No                      |
-| Route with TTL                                                  | Yes                                               | No                      |
-| Plug-in hot loading                                             | Yes                                               | No                      |
-| Custom LB and route                                             | Yes                                               | No                      |
-| REST API <--> gRPC transcoding                                  | Yes                                               | No                      |
-| Tengine                                                         | Yes                                               | No                      |
-| MQTT                                                            | Yes                                               | No                      |
-| Configuration effective time                                    | Event driven, < 1ms                               | polling, 5 seconds      |
-| Dashboard                                                       | Yes                                               | No                      |
-| IdP                                                             | Yes                                               | No                      |
-| Configuration Center HA                                         | Yes                                               | No                      |
-| Speed limit for a specified time window                         | Yes                                               | No                      |
-| Support any Nginx variable as routing condition                 | Yes                                               | No                      |
+| **Features**                                                    | **Apache APISIX**                                        | **Kong**                |
+| :-------------------------------------------------------------- | :------------------------------------------------------- | :---------------------- |
+| Belongs to                                                      | Apache Software Foundation                               | Kong Inc.               |
+| Tech Architecture                                               | Nginx + etcd                                             | Nginx + postgres        |
+| Communication channels                                          | Mail list, Wechat group, QQ group, GitHub, Slack, meetup | GitHub, freenode, forum |
```

Comment:
Hi it might be a good idea to align the last vertical bar 🤣  I guess some IDE plugin could do that